### PR TITLE
MWPW-137345 | [For Main] Interactive container block for CC only 

### PR DESCRIPTION
**Pull request by drashti1712**
* Implemented the base Interactive Marquee container for the upcoming variant - Genfill and Firefly
* Figma: https://www.figma.com/file/L1YpIBovlcXffNVvOePgA9/Interactive-Marquee-KR?type=design&node-id=56-1516&mode=design&t=URlCADx9xEuFjjuk-0

Resolves: [MWPW-138595](https://jira.corp.adobe.com/browse/MWPW-138595)

**Test URLs:**
- Before: https://main--cc--adobecom.hlx.page/drafts/drashti/MWPW-137345/container/?martech=off
- After: https://interactive-container--cc--drashti1712.hlx.page/drafts/drashti/MWPW-137345/container/?martech=off

### CODE REVIEW

**Review by suhjainadobe — COMMENTED** [review]
Could you make css changes specificifically for your block only as it will affect other interactive blocks as well.

File: creativecloud/blocks/interactive-marquee/interactive-marquee.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  const mediaElements = foreground.querySelectorAll(':scope > div:not([class])');
+  const media = mediaElements[0];
+  if (media) {
+    const interactiveBox = createTag('div', { class: 'interactive-container' });
+    interactiveBox.appendChild(media);
+    media.classList.add('media');
+    foreground.appendChild(interactiveBox);
+    const childNodes = media.querySelectorAll('p');
+    [...childNodes].forEach(async (child) => {
+      const video = child.querySelector('video, a[href*=".mp4"]');
```
Comment (Ruchika4) [inline creativecloud/blocks/interactive-marquee/interactive-marquee.js:75]: I think we can remove this line of code as we only need images to be decorated

File: creativecloud/blocks/interactive-marquee/interactive-marquee.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      const image = child.querySelector('img');
+      if (image && !video) {
```
Comment (Ruchika4) [inline creativecloud/blocks/interactive-marquee/interactive-marquee.js:77]: I believe we can just have a check for if(image) here

**Review by Ruchika4 — CHANGES_REQUESTED** [review]

**Review by aishwaryamathuria — APPROVED** [review]

**Comment by salonijain3** [comment]
@drashti1712 you can raise these changes with the main branch - interactive for genfill variant

**Pull request closed without merging**